src/locale/locales/fr/messages.po

```diff
@@ -2110,11 +2107,11 @@ msgstr "Échec de l’envoi de l’appel, veuillez réessayer."
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:180
 msgid "Failed to toggle thread mute, please try again"
-msgstr ""
+msgstr "Échec de basculer la masquage du fil de discussion, veuillez réessayer"
```

Comment:
(It's verbose, but I can't find a better way to write it. `Basculer` doesn't really map to `toggle` well in this context, imho.)

```suggestion
msgstr "Échec de l’activation ou désactivation du masquage du fil de discussion, veuillez réessayer"
```